spec/fixtures/addons/buttons.scss

```diff
@@ -1,17 +1,17 @@
 @import "setup";
 
-#{$all-button-inputs} {
-  color: #ff0000;
+#all-button-inputs {
```

Comment:
Can we just go ahead and use class selectors here and for all future tests. Better aligns with how we typically author and Hound will be a happy camper.